recipes/newrelic/apm/php/debian.yml

```diff
@@ -208,19 +208,22 @@ install:
             read -r APPLICATION_RESTART
             APPLICATION_RESTART=${APPLICATION_RESTART:-y}
 
-            echo "$APPLICATION_NAME $APPLICATION_RESTART" > "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}"
+            echo "$APPLICATION_NAME" > "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}"
+            echo "$APPLICATION_RESTART" >> "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}"
           fi
 
           # Interactive install with NEW_RELIC_APPLICATION_NAME envar set, uses envar and also defaults to service restart
           if [[ "{{.NEW_RELIC_ASSUME_YES}}" != "true" && ! -z $NEW_RELIC_APPLICATION_NAME ]]; then
-            echo "$NEW_RELIC_APPLICATION_NAME y" > "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}"
+            echo "$NEW_RELIC_APPLICATION_NAME" > "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}"
+            echo "y" >> "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}"
           fi
 
     verify_continue:
       cmds:
         - |
           if [[ "{{.NEW_RELIC_ASSUME_YES}}" != "true" ]]; then
-            read -r APPLICATION_NAME APPLICATION_RESTART < "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}"
+            read -r APPLICATION_NAME < "{{.TMP_INSTALL_DIR}}/{{.USER_INPUT}}"
```

Comment:
@rthorn-nr why don't we just consolidate the step user_input with verify_continue, and avoid the writing/reading from a temp file?

Reply:
Great idea @Julien4218 , I'll take a look at that this afternoon

Reply:
Actually @Julien4218 these are used in a later task when writing to newrelic.ini, so we would need to collapse all of those tasks to be able to scope them to a single task or write them to a temp file anyway. I'm trying to find alternatives to this but don't want to spend too much time on it. 

This strategy of using a temp file was merged a few weeks ago in https://github.com/newrelic/open-install-library/pull/877

Reply:
Ok, I didn't see it was used somewhere else. Something to change later then as we're looking into removing our dependency on go tasks.